src/ParallelAlgorithms/DiscontinuousGalerkin/InitializeMortars.hpp

```diff
@@ -118,6 +118,7 @@ struct InitializeMortars {
     const auto& interface_meshes =
         db::get<domain::Tags::Interface<domain::Tags::InternalDirections<dim>,
                                         domain::Tags::Mesh<dim - 1>>>(box);
+    const auto quadrature = db::get<domain::Tags::Mesh<dim>>(box).quadrature(0);
```

Comment:
For my education (nothing to do with this PR other than making me wonder about it...):
- why is `Tags::Mesh` in domain, when `Mesh` itself is in spectral? should the Tag have been moved when we moved the Mesh code?
- why is `Tags::Quadrature` in evolution, when the Quadrature itself if also in spectral?

Reply:
- I don't remember. Mesh used to be in `Domain`, but I forget why we moved it(probably cyclic deps or something depended on Mesh but we didn't want it to depend on Domain). 
- I was only planning on using it in the evolution code. Should be moved if we plan on using it more (e.g. elliptic code). We don't really have (any?) good rules for tags.